tools/ci/cibuild.sh

```diff
@@ -224,7 +224,7 @@ function xtensa-esp32-gcc-toolchain {
     esac
   fi
   xtensa-esp32-elf-gcc --version
-  pip3 install esptool
+  pip3 install esptool==3.0
```

Comment:
```suggestion
  pip3 install esptool==3.3.1
```

Version 3.0 does not provide support for newer chips like **ESP32-C3**  and **ESP32-S3**.

Reply:
Thanks for catching that!